package.json

```diff
@@ -80,5 +86,6 @@
       "node-tests",
       "tests"
     ]
-  }
+  },
+  "homepage": "https://trentmwillis.github.io/ember-exam"
```

Comment:
should this be `https://ember-cli.github.io/ember-exam`

Reply:
Yes. Of course unrelated, but I can update.